<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Dini1912" ignored="1" name="Test" tests="1" failures="1" timestamp="28 Jul 2023 11:46:07 GMT" time="1.213" errors="0">
  <testcase name="addtocartpage" time="1.198" classname="Homepage.loginpageTest">
    <failure type="java.lang.AssertionError" message="expected [Thank you for your order!
Your order has been dispatched, and will arrive just as fast as the pony can get there!
Back Home] but found [Thank you for your order!
Your order has been dispatched, and will arrive just as fast as the pony can get there!
Back Home]">
      <![CDATA[java.lang.AssertionError: expected [Thank you for your order!
Your order has been dispatched, and will arrive just as fast as the pony can get there!
Back Home] but found [Thank you for your order!
Your order has been dispatched, and will arrive just as fast as the pony can get there!
Back Home]
	at Homepage.loginpageTest.addtocartpage(loginpageTest.java:54)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- addtocartpage -->
  <testcase name="login" time="0.0" classname="Homepage.loginpageTest">
    <ignored/>
  </testcase> <!-- login -->
</testsuite> <!-- Test -->
